diff -ruNp starter-Starter-7.0.orig/StartProcessThread.cpp starter-Starter-7.0/StartProcessThread.cpp
--- starter-Starter-7.0.orig/StartProcessThread.cpp	2018-02-09 10:08:57.000000000 +0100
+++ starter-Starter-7.0/StartProcessThread.cpp	2018-12-10 10:04:49.556014364 +0100
@@ -222,7 +222,7 @@ void StartProcessThread::start_process(N
 #endif
 				// Close standard out
 				close(1);
-                open("/dev/null", O_RDWR | O_CREAT, 0664);
+//                open("/dev/null", O_RDWR | O_CREAT, 0664);
 
 				//	Close the stderr and re-open it on a log file.
                 close(2);
