diff -ruNp cppTango-9.3.3.orig/CMakeLists.txt cppTango-9.3.3/CMakeLists.txt
--- cppTango-9.3.3.orig/CMakeLists.txt	2019-03-29 09:49:12.000000000 +0100
+++ cppTango-9.3.3/CMakeLists.txt	2019-10-21 22:32:29.450535439 +0200
@@ -32,9 +32,6 @@ include_directories(${CMAKE_CURRENT_BINA
 #source code
 add_subdirectory("log4tango")
 add_subdirectory("cppapi")
-if(NOT WIN32)
-    add_subdirectory("cpp_test_suite")
-endif(NOT WIN32)
 
 if(WIN32)
     include(configure/cmake_win.cmake)
