diff -ruNp git.orig/setup.py git/setup.py
--- git.orig/setup.py	2019-11-27 14:30:22.870236575 +0100
+++ git/setup.py	2019-11-27 14:31:44.294509389 +0100
@@ -369,31 +369,7 @@ def setup_args():
 
     # special boost-python configuration
 
-    BOOST_ROOT = os.environ.get('BOOST_ROOT')
-    boost_library_name = 'boost_python'
-    if BOOST_ROOT is None:
-        if CONDA:
-            suffix = ''
-        elif DEBIAN:
-            suffix = "-py{v[0]}{v[1]}".format(v=PYTHON_VERSION)
-            boost_library_name += suffix
-        elif REDHAT:
-            if PYTHON3:
-                boost_library_name += '3'
-        elif GENTOO:
-            suffix = "-{v[0]}.{v[1]}".format(v=PYTHON_VERSION)
-            boost_library_name += suffix
-    else:
-        inc_dir = os.path.join(BOOST_ROOT, 'include')
-        lib_dirs = [os.path.join(BOOST_ROOT, 'lib')]
-        if IS64:
-            lib64_dir = os.path.join(BOOST_ROOT, 'lib64')
-            if os.path.isdir(lib64_dir):
-                lib_dirs.insert(0, lib64_dir)
-
-        directories['include_dirs'].append(inc_dir)
-        directories['library_dirs'].extend(lib_dirs)
-
+    boost_library_name = 'boost_python' + "{v[0]}{v[1]}".format(v=PYTHON_VERSION)
     directories['libraries'].append(boost_library_name)
 
     # special numpy configuration
